src/Raven.Server/Documents/Handlers/Admin/AdminMemoryHandler.cs

```diff
@@ -2,23 +2,54 @@
 using System.Runtime;
 using System.Threading.Tasks;
 using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
 using Raven.Server.Web;
+using Sparrow;
+using Sparrow.Json;
+using Sparrow.LowMemory;
 
 namespace Raven.Server.Documents.Handlers.Admin
 {
     public class AdminMemoryHandler : RequestHandler
     {
         [RavenAction("/admin/memory/gc", "GET", AuthorizationStatus.Operator)]
-        public Task CollectGarbage()
+        public async Task CollectGarbage()
         {
             var loh = GetBoolValueQueryString("loh", required: false) ?? false;
 
             if (loh)
                 GCSettings.LargeObjectHeapCompactionMode = GCLargeObjectHeapCompactionMode.CompactOnce;
 
+            var memoryBefore = AbstractLowMemoryMonitor.GetManagedMemoryInBytes();
+            var startTime = DateTime.UtcNow;
+
             GC.Collect(GC.MaxGeneration, GCCollectionMode.Forced);
 
-            return Task.CompletedTask;
+            var actionTime = DateTime.UtcNow - startTime;
+            var memoryAfter = AbstractLowMemoryMonitor.GetManagedMemoryInBytes();
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                writer.WriteStartObject();
+
+                writer.WritePropertyName("BeforeGC");
```

Comment:
isn't that the same as in /admin/debug/memory/gc ? FYI, we are using there GC.GetGCMemoryInfo, which probably is less error prone?